domino-extended-api/api.py

```diff
@@ -25,8 +29,7 @@
 
 logger = logging.getLogger("extended-api")
 app = Flask(__name__)
-
-
+app.register_blueprint(domsed_api)
```

Comment:
Probably in the next version we should break up the original api endpoints into two categories-
1. Domino API
2. AutoshutdownRules API